### PR TITLE
Return error if CSV/JSON don't have a name column

### PR DESCRIPTION
**Pull request by nachoiacovino**
Also added some style changes to airdrop/snapshot upload

### CODE REVIEW

File: contract-ui/tabs/nfts/components/batch-lazy-mint.tsx [hunk 1]
```diff
@@ -92,6 +87,8 @@ export const BatchLazyMint: React.FC<BatchLazyMintProps> = ({
 
   const mergedData = useMergedData(csvData, jsonData, imageFiles, videoFiles);
 
+  const invalidFiles = (csvData || jsonData) && mergedData.length === 0;
```
Comment (jnsdls) [inline contract-ui/tabs/nfts/components/batch-lazy-mint.tsx:90]: I assume `mergedData.length` will be `0` in the case that there is no name field?

File: contract-ui/tabs/nfts/components/upload-step.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ export const UploadStep: React.FC<UploadStepProps> = ({
               cursor="pointer"
               bg={noFile ? "red.200" : "inputBg"}
               _hover={{
-                bg: "inputBgHover",
+                bg: noFile ? "red.200" : "inputBgHover",
```
Comment (jnsdls) [inline contract-ui/tabs/nfts/components/upload-step.tsx:40]: is this true for *no* file or is it based on *there might be files but they are invalid?

Comment (jnsdls) [reply contract-ui/tabs/nfts/components/upload-step.tsx:40]: _just asking about var naming_

File: contract-ui/tabs/nfts/components/upload-step.tsx [hunk 3]
```diff
@@ -49,16 +49,21 @@ export const UploadStep: React.FC<UploadStepProps> = ({
...
-                  <Heading as={Text} size="label.md" color="gray.600">
+                  <Heading
+                    as={Text}
+                    size="label.md"
+                    lineHeight={1.2}
+                    color={noFile ? "red.500" : "gray.600"}
+                  >
                     {noFile
-                      ? "No CSV or valid JSON file found, please try again"
+                      ? "No valid CSV/JSON file found, make sure your NFTs have a `name` column and try again"
```
Comment (jnsdls) [inline contract-ui/tabs/nfts/components/upload-step.tsx:66]:
```suggestion
                      ? `No valid CSV or JSON file found. Please make sure your NFT metadata includes at least a "name" field and try again.`
```

seems a bit easier to understand?

File: contract-ui/tabs/claim-conditions/components/snapshot-upload.tsx [hunk 4]
```diff
@@ -184,17 +186,29 @@ export const SnapshotUpload: React.FC<SnapshotUploadProps> = ({
...
+                          Drop the files here
+                        </Heading>
+                      ) : (
+                        <Heading
+                          as={Text}
+                          size="label.md"
+                          color={noCsv ? "red.500" : "gray.600"}
+                        >
+                          {noCsv
+                            ? "No valid CSV file found, make sure you have an address column."
```
Comment (jnsdls) [inline contract-ui/tabs/claim-conditions/components/snapshot-upload.tsx:207]:
```suggestion
                            ? `No valid CSV file found, make sure your CSV includes the "address" column.`
```

File: contract-ui/tabs/nfts/components/batch-lazy-mint.tsx (hunk 1 above)
Comment (nachoiacovino) [reply contract-ui/tabs/nfts/components/batch-lazy-mint.tsx:90]: yes ^

File: contract-ui/tabs/nfts/components/upload-step.tsx (hunk 2 above)
Comment (nachoiacovino) [reply contract-ui/tabs/nfts/components/upload-step.tsx:40]: it's actually `noFileOrOnlyInvalidFiles` but yeah

**Pull request merged**